### ISSUE TITLE
Unable to open visual studio by double click

### ISSUE THREAD

**Issue by yanek** [modesttree/Projeny#81]
Hello,

I'm just installing Projeny (0.3.11 binary release) for the first time, and everything works as intended except for the fact that I cannot open any C# script by double clicking the file (in Unity) or an error pointing to it.

I followed [the documentation](https://github.com/modesttree/Projeny#7---more-intelligent-visual-studio-solution-generation) to configure everything, and my `VisualStudioIdePath` is correctly set to `'E:/Bin/Microsoft Visual Studio/2017/Community/Common7/IDE/devenv.exe'`.

Here's the message I get in the console:
```
Started OpenInVisualStudio with arguments: D:\Projeny\UnityProjects\TestProj\TestProj-Windows\Assets\LF_Main\UI\Scripts\Inventory.cs 0
Determined Project = TestProj, Platform = Windows
'Image Name'
See PrjLog.txt for more details
Operation completed with errors.  Took 0.0 seconds.
Error occurred!  Unable to open visual studio.
```

I checked PrjLog.txt multiple times, but it stays empty or contains messages regarding other operations.
Note that I can open VS2017 by going `Projeny -> Packages Manager... -> Open Solution`, and it works just fine.

Any help would be welcome.

**Comment by Gravetow** [comment]
Hello Yanek!

Did you follow these instructions as well?
`
If you want to use the Projeny-generated visual studio solution as your primary editor, you can also set up Unity to use it by default. You can do this by doing the following:

Open Unity and go to Edit -> Preferences -> External Tools
Click the dropdown next to External Script Editor
Click Browse
Navigate to the install directory of Projeny (by default this is at C:\Program Files (x86)\Projeny) and select Bin/PrjOpenInVisualStudio.bat (note that you may have to select AllFiles in the bottom right dropdown)
Next to External Script Editor Args type "$(File)" "$(Line)"
`
Please make sure the External Script Editor Args type are set properly.

Other then that you might want to try to define the UnityExePath as well in your Projeny.yaml.
I had some issues getting the whole setup to work in the beginning too and addind the UnityExePath helped in my case. 
I'm not sure if it will for you though!

**Comment by yanek** [comment]
Yep, I followed every instructions. I also tried to start `.../Projeny/Bin/PrjOpenInVisualStudio.bat` and even `.../Projeny/Bin/Data/OpenInVisualStudio.exe` directly from the Windows command line with valid arguments, and I got the same error message.

I also had some issues getting Projeny to work; I was not able to link packages from the asset store, and had to add `UnityExePath` early on the setup process. So everything should be fine on this side.

Here's the full Proneny.yaml file I am currently using: 
```yaml
PathVars:
    UnityExePath: 'E:/Bin/Unity/Editor/Unity.exe'
    UnityProjectsDir: '[ConfigDir]/UnityProjects'
    SharedUnityPackagesDir: '[ConfigDir]/UnityPackages'
    LogPath: '[ConfigDir]/PrjLog.txt'
    VisualStudioIdePath: 'E:/Bin/Microsoft Visual Studio/2017/Community/Common7/IDE/devenv.exe'
```

Could it be because Unity, VS2017 and Projeny are installed on a drive (E) while my projects and packages are on another one (D)?
